Makefile.am

```diff
@@ -231,48 +250,54 @@ PKCS11_SO_PIN ?= 76543210
 PKCS11_USER_PIN ?= 01234567
 PKCS11_VHSM_PIN ?= 1234567890
 
+if AIX
+PKCSLIB = '@libdir@/opencryptoki/libopencryptoki.a(libopencryptoki.so.0)'
+else
+PKCSLIB ?= @libdir@/opencryptoki/libopencryptoki.so
+endif
+
 ci-prepare:
-	killall -HUP pkcsslotd || true
+	$(KILLALL) -HUP pkcsslotd || true
 	${srcdir}/testcases/ciconfig.sh "$(sysconfdir)/opencryptoki" "$(sysconfdir)/opencryptoki" "$(sysconfdir)/opencryptoki" "${srcdir}/testcases/test_combined_extract.slots"
 	@sbindir@/pkcsslotd
 	@sbindir@/pkcsstats --reset-all
-	for slot in `awk '/slot (.*)/ { print $$2; }' $(sysconfdir)/opencryptoki/opencryptoki.conf`; do @sbindir@/pkcsconf -c $$slot -t | grep "Flags:" | grep -q TOKEN_INITIALIZED || PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so ${srcdir}/testcases/init_token.sh $$slot; done
-#	cd ${srcdir}/testcases && PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCS11_VHSM_PIN=$(PKCS11_VHSM_PIN) PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so ./init_vhsm.exp 42
+	for slot in `awk '/^slot (.*)/ { print $$2; }' $(sysconfdir)/opencryptoki/opencryptoki.conf`; do @sbindir@/pkcsconf -c $$slot -t | grep "Flags:" | grep -q TOKEN_INITIALIZED || PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCSLIB=$(PKCSLIB) ${srcdir}/testcases/init_token.sh $$slot; done
+#	cd ${srcdir}/testcases && PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCS11_VHSM_PIN=$(PKCS11_VHSM_PIN) PKCSLIB=$(PKCSLIB) ./init_vhsm.exp 42
 #	echo "VHSM_MODE" >> "$(sysconfdir)/opencryptoki/ep11tok42.conf"
 
-installcheck-local: all
-	killall -HUP pkcsslotd || true
+installcheck-local: all check
+	$(KILLALL) -HUP pkcsslotd || true
 	@sbindir@/pkcsslotd
 	if test ! -z ${PKCS11_TEST_USER}; then				\
 		chmod 777 ${srcdir}/testcases &&			\
 		cd ${srcdir}/testcases &&                               \
-		su ${PKCS11_TEST_USER} -s /bin/bash -c "PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so /bin/bash ./ock_tests.sh || true"; \
+		su ${PKCS11_TEST_USER} -s /bin/bash -c "PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCSLIB=$(PKCSLIB) /bin/bash ./ock_tests.sh || true"; \
 	else								\
 		cd ${srcdir}/testcases && 				\
-		PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCSLIB=@libdir@/opencryptoki/libopencryptoki.so /bin/bash ./ock_tests.sh || true; \
+		PKCS11_SO_PIN=$(PKCS11_SO_PIN) PKCS11_USER_PIN=$(PKCS11_USER_PIN) PKCSLIB=$(PKCSLIB) ./ock_tests.sh || true; \
 	fi
-	killall -HUP pkcsslotd
+	$(KILLALL) -HUP pkcsslotd
 
 ci-installcheck: ci-prepare installcheck
-	killall -HUP pkcsslotd || true
+	$(KILLALL) -HUP pkcsslotd || true
```

Comment:
The commit message still mentions:

_A new target has been added that only runs the utility check (p11sak, pkcsconf)
for easier development-time testing. It has also been added to installcheck
so that the existing checks continue to work._

I guess you have removed that some time ago, so maybe you need to update the commit message accordingly.